ModelCatalogueFormsPlugin/grails-app/assets/javascripts/mc/core/forms/templates/formItem.tpl.html

```diff
@@ -1,5 +1,5 @@
 <div class="col-md-12 metadata-form-item-content">
-  <div class="row">
+  <div class="row" ng-show="view.handlesKey('http://forms.modelcatalogue.org/item#phi')">
```

Comment:
looks good, just as we discussed yesterday I would prefer ng-if instead of ng-show.

see e.g. http://www.codelord.net/2015/07/28/angular-performance-ng-show-vs-ng-if/